src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Missing header comment. All classes, interfaces and non trivial methods should have javadoc format header comments.